tests/integration/individual_response/test_individual_response.py

```diff
@@ -0,0 +1,375 @@
+
+    def _add_no_household_members(self):
+        self.get("questionnaire/primary-person-list-collector/")
+        self.post({"you-live-here": "No"})
+        self.post({"anyone-else": "No"})
+        self.get("questionnaire/")
+
+    def _add_primary(self):
+        self.get("questionnaire/primary-person-list-collector/")
+        self.post({"you-live-here": "Yes"})
+        self.post({"first-name": "Marie", "last-name": "Day"})
+        self.post({"anyone-else": "No"})
+        self.get("questionnaire/")
+
+    def _add_primary_and_household(self):
+        self.get("questionnaire/primary-person-list-collector/")
+        self.post({"you-live-here": "Yes"})
+        self.post({"first-name": "Marie", "last-name": "Day"})
+        self.post({"anyone-else": "Yes"})
+        self.post({"first-name": "John", "last-name": "Doe"})
+        self.post({"anyone-else": "No"})
+        self.get("questionnaire/")
+
+    def _add_household_no_primary(self):
+        self.get("questionnaire/primary-person-list-collector/")
+        self.post({"you-live-here": "No"})
+        self.post({"anyone-else": "Yes"})
+        self.post({"first-name": "Marie", "last-name": "Day"})
+        self.post({"anyone-else": "No"})
+        self.get("questionnaire/")
+
+    def _request_individual_response(self):
+        self._add_household_no_primary()
+        self.post()
+        self.get(self.individual_response_link)
+        self.post()
+        self.post({"individual-response-how-answer": "Post"})
+        self.post(
+            {
+                "individual-response-post-confirm-answer": "Yes, send the access code by post"
+            }
+        )
+        self.post()
+
+    def test_401_after_signout(self):
+        # Given the hub is enabled
+        # And I add a household member
+        self._add_household_no_primary()
+
+        # When I sign out and navigate to the individual response page
+        self.get(self.sign_out_link)
+        self.get("/individual-response")
+
+        # Then I should see the signed out page
+        self.assertInBody("Your session has expired")
+
+    def test_404_invalid_list_item_id(self):
+        # Given the hub is enabled
+        self._add_household_no_primary()
+        # When I use an invalid id in an individual response url
+        self.get("/")
+        self.get(f"individual-response/not-an-id/how")
+        self.assertStatusCode(404)
+
+    def test_how_route_redirects_signed_out(self):
+        # Given the hub is enabled
+        # And I add a housheold member
+        self._add_household_no_primary()
+        self.post()
+
+        # When I sign out and navigate to the individual response page
+        individual_response_link = self.individual_response_link
+        self.get(individual_response_link)
+        self.post()
+        self.get(self.sign_out_link)
+        self.get(individual_response_link)
+
+        # Then I should see the signed out page
+        self.assertInBody("Your session has expired")
```

Comment:
This should assert for the HTTP status code, rather than the text (we wouldn't want the text of these pages changing to break these tests). 

This applies to the other checks like this in the following tests e.g. "Page not found" -> 404.